pyracing/helpers.py

```diff
@@ -30,6 +30,7 @@ def now_five_min_floor():
 
 
 def parse_encode(string):
-    if not string:
+    if not type(string) is str:
         return None
```

Comment:
I know you didn't change this, but we should probably return an empty string here as whatever is using this likely is going to use the response as a string would be used. That should make it a bit safer.

Reply:
```suggestion
    return ''
```
Okay, that makes more sense.